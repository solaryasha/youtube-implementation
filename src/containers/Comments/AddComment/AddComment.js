import React from 'react';
import './AddComment.scss';
import { Image, Form, TextArea } from 'semantic-ui-react';

const AddComment = () => (
    <div className='add-comment'>
        <Image className='user-image' src='https://via.placeholder.com/48x48' circular />
        <Form>
            <Form.Field control={TextArea} autoHeight placeholder='Add a public comment'/>
        </Form>
    </div>
)

export default AddComment;