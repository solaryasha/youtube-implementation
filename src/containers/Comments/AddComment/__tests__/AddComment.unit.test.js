import React from 'react';
import { shallow } from 'enzyme';
import AddComment from '../AddComment';


describe('AddComment', () => {
    test('renders without props', () => {
        const wrapper = shallow(<AddComment />);

        expect(wrapper).toMatchSnapshot();
    })

})