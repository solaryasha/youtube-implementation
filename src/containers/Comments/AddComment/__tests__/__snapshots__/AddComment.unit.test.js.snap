// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddComment renders without props 1`] = `
<div
  className="add-comment"
>
  <Image
    as="img"
    circular={true}
    className="user-image"
    src="https://via.placeholder.com/48x48"
    ui={true}
  />
  <Form
    as="form"
  >
    <FormField
      autoHeight={true}
      control={[Function]}
      placeholder="Add a public comment"
    />
  </Form>
</div>
`;
